app/services/bulk_upload/sales/year2023/row_parser.rb

```diff
@@ -1334,4 +1335,8 @@ def validate_buyer1_economic_status
       errors.add(:field_35, "Buyer 1 cannot be a child under 16.")
     end
   end
+
+  def downcase(str)
+    downcase_first_letter(str)
+  end
```

Comment:
Add a more concisely named private method to each parser, using the downcase_first_letter method from the helper.